_posts/0000-01-02-JayneMarlowe.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
is this line 5 or a comment? I don't seem to be able to change line 5 as the directions tell me to do.
I sincerely hope no one is actually seeing this!